cnf-certification-test/certification/suite.go

```diff
@@ -53,7 +53,19 @@ var _ = ginkgo.Describe(common.AffiliatedCertTestKey, func() {
 	ginkgo.It(testID, ginkgo.Label(tags...), func() {
 		testContainerCertificationStatus(&env)
 	})
-
+	var _ = ginkgo.Describe("extra", func() {
```

Comment:
Why do an entire "extra" `Describe` block?

Reply:
we add this extra to not run this test when we run it with the label of certification

Reply:
I'm not sure I follow.  I think you just need to set labels that get gathered from `GetGinkgoTestIDAndLabels`.

Reply:
I agree with Brandon. This new"Describe" block is not needed.